crates/subspace-networking/Cargo.toml

```diff
@@ -23,11 +23,15 @@ event-listener-primitives = "2.0.1"
 futures = "0.3.21"
 generic-array = "0.14.5"
 hex = "0.4.3"
+lru = "0.7.7"
 nohash-hasher = "0.2.0"
 once_cell = "1.12.0"
+parity-db = "0.3.14"
 parity-scale-codec = "3.1.2"
 parking_lot = "0.12.0"
 pin-project = "1.0.10"
+serde = { version = "1.0.137", features = ["derive"] }
+serde_json = "1.0.81"
```

Comment:
Dependencies were updated in the meantime, we want to ideally keep them the same everywhere:
```suggestion
lru = "0.7.8"
nohash-hasher = "0.2.0"
once_cell = "1.12.0"
parity-db = "0.3.14"
parity-scale-codec = "3.1.2"
parking_lot = "0.12.0"
pin-project = "1.0.10"
serde = { version = "1.0.139", features = ["derive"] }
serde_json = "1.0.82"
```